docs/advanced-features/custom-server.md

```diff
@@ -53,10 +53,13 @@ app.prepare().then(() => {
       res.statusCode = 500
       res.end('internal server error')
     }
-  }).listen(port, (err) => {
-    if (err) throw err
-    console.log(`> Ready on http://${hostname}:${port}`)
   })
+    .once('error', (err) => {
+      throw err
```

Comment:
I expect the semantics to be the same as with `.catch()` so this would be a NOOP.
Probably better to use `console.error` + `throw`

Reply:
> I expect the semantics to be the same as with .catch() so this would be a noop.

It raises an `uncaughtException` which I believe has different semantics depending on node.js version.

Reply:
Oh :+1: